docs/get-started/lotus/bootstrappers.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Running Bootstrapper Nodes
+description: 'Running your own bootstrapper nodes'
+breadcrumb: 'Bootstrapper nodes'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+### Bootstrappers
+
+Lotus daemons may be configured to act as network bootstrappers. Bootstrappers act as the initial point of
+contact for other lotus daemons to find peers.
+
+::: tip
+Unless you are running a private test network, it is not usuaally necessary for to run a network bootstrapper node.
+:::
+
+### Configuring a Bootstrapper node
+
+Because bootstrapper nodes handle somewhat different traffic than a typical lotus daemon, we recommend using
+the following configuration. This configuration will setup the node to act as a bootstrapper for 
+
+```toml
+  [API]
+    ListenAddress = "/ip4/0.0.0.0/tcp/1234/http"
+  [Libp2p]
+    ListenAddresses = ["/ip4/0.0.0.0/tcp/1347"]
+    ConnMgrLow = 400
+    ConnMgrHigh = 500
+    ConnMgrGrace = "5m0s"
+  [Pubsub]
+    IPColocationWhitelist = ["x.x.x.x/x"]
```

Comment:
This configuration isn't required, but could be documented separately. It's use is rather special cased that I'm not sure if we need to document here.